app/controllers/projects_controller.rb

```diff
@@ -1,4 +1,13 @@
 class ProjectsController < ApplicationController
+  def new
+  end
+
+  def index
+  end
+
   def show
   end
+
+  def edit
+  end
```

Comment:
`new` and `edit` should be removed from this controller since those actions will probably be handled in `Admins::ProjectsController` when it gets written to make sure authorization is done properly.